Dockerfile

```diff
@@ -5,7 +5,7 @@ WORKDIR /app
 
 COPY . .
 
-RUN apk add build-base libffi-dev
+RUN apk add build-base libffi-dev postgresql-dev
```

Comment:
Does this package need to be installed then?

Reply:
If you want to support pgsql, yes, this is postgresql client that used by psycopg2
Reference: [Error: pg_config executable not found when installing psycopg2 on Alpine in Docker](https://stackoverflow.com/questions/46711990/error-pg-config-executable-not-found-when-installing-psycopg2-on-alpine-in-dock)